R/palette.R

```diff
@@ -16,11 +16,13 @@ default_palette <- function() {
 
 #' All-purpose Synapse palette
 #'
-#' Return max of 10 synapse colors.
+#' Return max of 15 synapse colors.
 #' @export
 syn_palette <- function() {
```

Comment:
love this implementation of palette :) 
Some time ago there was a concerted effort to consolidate Sage branding colors. So adding this for consideration: https://sage-bionetworks.github.io/sagethemes/index.html

Reply:
Ha, thanks, didn't know about this, good to chat with Abby about it some time, maybe.